collector/types/types.go

```diff
@@ -163,4 +164,5 @@ type ValidatorInfo struct {
 	Moniker        string
 	CommissionRate float64
 	Jailed         bool
+	Tokens         *big.Int
```

Comment:
isnt a tokens represent a total amount of delegated coins? Should not we use instead m.apiClient.Staking.GetStakingDelegatorsDelegatorAddrDelegations endpoint to get our share?